src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,148 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private Object[] arrayMain = new Object[10];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
class fields have default values